### PR TITLE
[Login Page]: Adjust Login Page to mobile

### PR DESCRIPTION
**Pull request by meczajkowski**
- The login page has been adjusted for mobile compatibility.
- Redirect buttons to the login page have been added to the landing page.

TODO
- Add login buttons to the /profiles page.
- Implement a confirmation message after successfully joining as a hunter.

image
image

### CODE REVIEW

**Comment by meczajkowski** [comment]
Ready for Review:

- Adjusted spacing.
- Implemented confirmation message display after sending the magic link.
- Added login buttons to the /profiles route.

File: src/app/(profile)/(components)/AppHeader/AppHeader.module.scss [hunk 1]
```diff
@@ -73,7 +73,7 @@
   justify-content: center;
   align-items: center;
   gap: 12px;
-  max-width: 420px;
+  // max-width: 420px;
 }
```
Comment (wawrzonnn) [inline src/app/(profile)/(components)/AppHeader/AppHeader.module.scss:78]: comment to remove?

File: src/components/Button/Button.module.scss [hunk 2]
```diff
@@ -10,7 +11,7 @@
   background-color: #7d54f1;
   border-radius: 2px;
   border: none;
-  outline: none;
+  // outline: none;
   font-size: 14px;
   font-style: normal;
```
Comment (wawrzonnn) [inline src/components/Button/Button.module.scss:16]: comment

**Review by wawrzonnn — APPROVED** [review]
GJ

**Pull request merged**